ngraph/frontend/onnx/frontend/src/place.hpp

```diff
@@ -13,6 +13,7 @@ namespace frontend {
 class PlaceInputEdgeONNX : public Place {
 public:
     PlaceInputEdgeONNX(const onnx_editor::InputEdge& edge, std::shared_ptr<onnx_editor::ONNXModelEditor> editor);
+    PlaceInputEdgeONNX(onnx_editor::InputEdge&& edge, std::shared_ptr<onnx_editor::ONNXModelEditor> editor);
```

Comment:
why onnx_editor is used in FE ONNX API?

CC @ilyachur is it OK?

Reply:
Why not? It is internal ONNX frontend API

Reply:
To end user is exposed only Place API defined in `frontend_manager/include/frontend_manager/place.hpp` and the Place ctors cannot be directly called by a user